### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sandra1525**
Added link to CNCF Meshery Community/signoff

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Zain-Muiz** [comment]
Heyy @Sandra1525 ! Great work here ! It would be great to have the community link there. Good work! 

Although I have one suggestion :
- I think you missed out to signoff while commiting.  [Check here](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin) to understand how it is done.

@warunicorn19 could you add more on how to fix it this time ?

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Find out more on the <a href="https://layer5.io/community#meshmate">Layer5 commu
 ✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">meeting recordings</a>.<br />
 ✔️ <em><strong>Access</strong></em> the <b>Community Drive</b> by completing a community <a href="https://layer5.io/newcomer">Member Form</a>.<br />
 ✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
-<br />
+✔️ <em><strong>Join Us</strong></em> on <a href="https://community.cncf.io/meshery-community/">  CNCF Meshery Community </a>.<br />
```
Comment (leecalcote) [inline README.md:40]:
```suggestion
✔️ <em><strong>RSVP</strong></em> to the <a href="https://community.cncf.io/meshery-community/">CNCF Meshery Community</a> meetings.<br />
```

**Review by Zain-Muiz — COMMENTED** [review]
LGTM! @leecalcote  or more than which, "CITY" .  xD.  
 
 What do we do about the DCO ? @warunicorn19 @leecalcote

**Comment by warunicorn19** [comment]
@leecalcote if you could please pass of the DCO and merge this.

**Comment by leecalcote** [comment]
@warunicorn19 👍

**Pull request merged**